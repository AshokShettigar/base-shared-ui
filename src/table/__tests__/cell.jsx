import { shallow } from 'enzyme';
import Cell from 'platform-common-ui/table/cell';

describe('table/cell', function() {
    it('should render the cell', function() {
        const wrapper = shallow(<Cell />);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render the cell with class name', function() {
        const wrapper = shallow(<Cell className="my-class">Some data</Cell>);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render the cell with data', function() {
        const wrapper = shallow(<Cell>Some data</Cell>);
        expect(wrapper).toMatchSnapshot();
    });
});
