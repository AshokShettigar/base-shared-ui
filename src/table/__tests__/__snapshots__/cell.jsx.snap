// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`table/cell should render the cell 1`] = `
<td
  className="cn-table__cell"
  data-component="Cell"
/>
`;

exports[`table/cell should render the cell with class name 1`] = `
<td
  className="cn-table__cell my-class"
  data-component="Cell"
>
  Some data
</td>
`;

exports[`table/cell should render the cell with data 1`] = `
<td
  className="cn-table__cell"
  data-component="Cell"
>
  Some data
</td>
`;
